src/zenml/integrations/kubernetes/orchestrators/kubernetes_orchestrator.py

```diff
@@ -422,6 +423,37 @@ def prepare_or_run_pipeline(
             )
             return
 
+        # We set some default minimum resource requests for the orchestrator pod
```

Comment:
Can we extract that maybe into a method on the `KubernetesPodSettings`? Like `apply_default_requests()`, in order to avoid having this code around multiple times?